src/main/java/se/sundsvall/notes/api/NotesResource.java

```diff
@@ -83,18 +86,21 @@ public ResponseEntity<Note> updateNote(
 	@ApiResponse(responseCode = "404", description = "Not found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	public ResponseEntity<Note> getNoteById(
-		@Parameter(name = "id", description = "Note ID", example = "b82bd8ac-1507-4d9a-958d-369261eecc15") @ValidUuid @PathVariable(name = "id", required = true) final String id) {
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
```

Comment:
Do we need the municipalityId-attribute here? 
Is there a risk that we will get two identical UUID:s on different municipalities?

Reply:
I can agree. At first I did not include municipalityId here and in delete method. I then looked how we did in SupportManagement and solved it like we do for errands. But maybe it's redundant. I added it more for consistens.